airflow/utils/db.py

```diff
@@ -1022,8 +1026,24 @@ def _validate_version_range(command, config, version_range):
             'Please provide Airflow version range with the format "old_version:new_version"'
         )
     lower, upper = version_range.split(':')
+
     if not REVISION_HEADS_MAP.get(lower) or not REVISION_HEADS_MAP.get(upper):
-        raise AirflowException('Please provide valid Airflow versions that has migrations between them.')
+        raise AirflowException('Please provide valid Airflow versions above 2.0.0.')
+    if REVISION_HEADS_MAP.get(lower) == 'No Migration' or REVISION_HEADS_MAP.get(upper) == 'No Migration':
```

Comment:
I'm not sure this is right. e.g. 2.2.1 -> 2.2.3 shouldn't show this message. Same with 2.1.2 -> 2.2.3.

Reply:
Yeah. I think they should repeat the hash from 2.2.0 and the message should be only printed if hash(lower) == hash(upper).